README.md

```diff
@@ -2,7 +2,12 @@
 
 # Project Name - portfolio-website
 
-**A portfolio website that runs smoothly on different devices and screen sizes, this is my second Microverse repository on GitHub.**
+**A portfolio website that runs smoothly on different devices and screen sizes(Current design is for mobile only), this is my second Microverse repository on GitHub.**
+
+**Preview**
+
+**![Mobile-View](https://user-images.githubusercontent.com/107482707/198753670-98f2317a-a211-4c52-950c-cca9711c328b.png)**
+
```

Comment:
Your readme is looking professional 😉, but it can be improved. To do so:
It will look better if you let only your project name, instead of Project Name - "project name"
![image](https://user-images.githubusercontent.com/42799579/198755503-86af8ad4-1ad9-4d9d-a3af-b067493a5692.png)

Kindly avoid empty sections:
![image](https://user-images.githubusercontent.com/42799579/198755563-906aa6d1-14af-4e01-bcae-fd64116d5cbd.png)
Instead, add simple and direct steps, like this:
![image](https://user-images.githubusercontent.com/42799579/198755593-015fc112-1fe5-466d-af03-08027f781a14.png)


Please add the corresponding link to your social media:
![image](https://user-images.githubusercontent.com/42799579/198755531-0b50a021-6e27-4170-813d-c6793e7563c8.png)